.circleci/deploy_to_kubernetes_live_0

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh -eu
+
+ROOT=$(dirname "$0")
+NAMESPACE="$1"
+NAMESPACE_DIR="$ROOT/../kubernetes_deploy/$NAMESPACE"
+WEBAPP_REPO="$2"
+NGINX_REPO="$3"
+
+if ! [ $NAMESPACE ] ; then
+
+  echo "usage: deploy_to_kubernetes namespace\n"
```

Comment:
_nitpick_ Inconsistent filename due to the rename. `$0` may be safer.

_newlines_
`echo "text\n"` does not behave the same way as on a Mac, it will need an extra `echo -e` argument for the newline to work:

```
$ docker run -ti ubuntu:trusty-20180712 echo "test\n"
test\n

$ docker run -ti ubuntu:trusty-20180712 echo -e "test\n"
test

```

🙋‍♂ `shellcheck` is really good to pick out these:

```
$ shellcheck demo.sh

In demo.sh line 3:
echo "asd\n"
     ^-----^ SC2028: echo may not expand escape sequences. Use printf.

For more information:
  https://www.shellcheck.net/wiki/SC2028 -- echo may not expand escape sequen...
```

Reply:
Good spots. These are more relevant to the other file, as this is the renamed existing one which will be going away soon. I'll bear them in mind for future work